pywbem/cim_operations.py

```diff
@@ -486,6 +487,32 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     the :attr:`last_raw_request` and :attr:`last_raw_reply` attributes
     of the connection object.
 
+    WBEMConnections can record the results of methods that interact with
+    a WBEM Server using either the :class:`~pywbem.LogOperationRecorder` (uses
```

Comment:
The link to class LogOperationRecorder does not resolve, because it is not included in any .rst file.

Reply:
Think added OK.